test/test_camera_capture.py

```diff
@@ -1,6 +1,92 @@
 import pytest
 
 
+def test_capture_2d_3d_one_2d_and_one_3d(shared_file_camera):
+    import zivid
+
+    acquisitions3d = [zivid.Settings.Acquisition()]
+    acquisitions2d = [zivid.Settings2D.Acquisition()]
+    settings = zivid.Settings(
+        acquisitions=acquisitions3d, color=zivid.Settings2D(acquisitions=acquisitions2d)
+    )
+
+    with shared_file_camera.capture_2d_3d(settings) as frame:
+        assert frame
+        assert isinstance(frame, zivid.frame.Frame)
```

Comment:
Could in this file test a bit more, such that the frame.settings matches what is set above, which would reduce chances of bugs in this code.

Reply:
Fixed